### PR TITLE
Orbit db CRUD implementation for payroll dapp

### PR DESCRIPTION
**Pull request by coderwithattitude**

### CODE REVIEW

File: contracts/DAIPayrollDB.sol [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        address admin;
+        Employees[] employee;
+    }
+ 
+    mapping (address => Oraganizations) orgs;
+
+    function addOrganisation(string memory name, address[] memory employees, uint[] memory wages) public {
+        require(employees.length == wages.length,"employee and wages array lengths dont match");
+        orgs[msg.sender].name = name;
+        orgs[msg.sender].admin = msg.sender;
```
Comment (adibas03) [inline contracts/DAIPayrollDB.sol:21]: this allows overwriting the orgs, without checking if it exists

File: contracts/DAIPayrollDB.sol [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        for(uint8 i = 0; i <= employees.length ; i++){
```
Comment (adibas03) [inline contracts/DAIPayrollDB.sol:22]: Advisable tohave a maximum number of acceptable array sizes

File: contracts/DAIPayrollDB.sol [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            orgs[msg.sender].employee[i].addr = employees[i];
+            orgs[msg.sender].employee[i].wages = wages[i];
+        }
+    }
+    
+    function addEmployee(address addr, uint wages) public {
+        require(msg.sender == orgs[msg.sender].admin,"address not valid");
+        uint count = orgs[msg.sender].employee.length;
+        orgs[msg.sender].employee[count++].addr = addr;
```
Comment (adibas03) [inline contracts/DAIPayrollDB.sol:31]: Does this work,
You should need to update length first, right?

**Comment by coderwithattitude** [comment]
@adibas03 kindly ignore the smart contract it is deprecated as we would be using orbitdb for storage.it is only there for refrence

**Review by adibas03 — APPROVED** [review]

**Comment by adibas03** [comment]
@coderwithattitude Then, it looks all good.
Ready to be merged

**Pull request merged**